server/terraform/project.tf

```diff
@@ -0,0 +1,41 @@
+variable "project_name" {
+  type = string
+}
+variable "billing_account" {
+  type = string
+}
+variable "region" {
```

Comment:
Should this be `location_id` to match the convention for Google Cloud? Or is this a convention for Terraform?